build.gradle

```diff
@@ -39,7 +39,7 @@ dependencies {
     implementation 'com.bc.zarr:jzarr:0.3.5'
     implementation 'info.picocli:picocli:4.2.0'
     implementation 'me.tongfei:progressbar:0.9.0'
-    implementation 'ome:formats-bsd:6.12.0'
+    implementation 'ome:formats-bsd:6.13.0'
     // be careful here, this is a newer version of org.json:json than formats-gpl uses
     implementation 'org.json:json:20190722'
```

Comment:
Now that this dependency has been upgraded in the Bio-Formats stack, do we want to look into dropping this dependency and consume the transitive one?

Reply:
That required switching to formats-gpl, since formats-bsd does not have a dependency on `org.json:json`. I'm fine with that, but can also revert a187d92 and just bump the `org.json:json` version if you prefer.